README.md

```diff
@@ -64,6 +64,7 @@ It's based on Rails 4 and Ruby 2.1.5.
 * `bin/quality` - runs rubocop, brakeman, rails_best_practices and bundle-audit for the app
 * `bin/ci` - should be used in the CI or locally
 * `bin/server` - to run server locally
+* `scss-lint <folder name>` - runs SCSS-Lint
```

Comment:
@rustemginiyatullin I believe `scss-lint` should be added to the `bin/quality` script.
No need to mention it here separately.